boot/freeldr/freeldr/arch/i386/i386vid.c

```diff
@@ -205,7 +204,8 @@ USHORT BiosIsVesaSupported(VOID)
 
     if (Regs.w.ax != 0x004F)
     {
-        ERR("VESA BIOS call failed\n");
+        WARN("VBE Get SuperVGA information function not supported (AL=0x%x) or failed (AH=0x%x)\n",
```

Comment:
These trace changes look fine. All the unrelated noise around it does not.

Reply:
Trace changes splitted to #291.